merlin-rxjava2/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest
+  package="com.novoda.merlin.rxjava2">
```

Comment:
We can use the single line here `<manifest package="com.novoda.merlin.rxjava" />`